src/tests/test_plot.py

```diff
@@ -112,19 +113,47 @@ def test_summary_stats_missing_file(chinook_db, ip_empty):
     assert 'No files found that match the pattern "data.csv"' in str(e)
 
 
+# Test internal plot function for data with nulls
 @pytest.mark.parametrize(
-    "cell, error_type, error_message",
+    "cell, error_check",
     [
         [
             "%sqlplot histogram --table data.csv --column age --table data.csv",
-            ValueError,
-            "Data contains NULLs",
+            None,
         ]
     ],
 )
-# Test internal plot function e.g.
-def test_internal_histogram_exception(tmp_empty, ip, cell, error_type, error_message):
-    Path("data.csv").write_text("name,age\nDan,33\nBob,19\nSheri,")
+def test_internal_histogram_null_support(tmp_empty, ip, cell, error_check):
+    # sheri, mick missing age
+    Path("data.csv").write_text(
+        "name,age\nDan,33\nBob,19\nSheri,\nVin,33\nMick,\nJay,33\nSky,33"
+    )
+    ip.run_cell("%sql duckdb://")
+    ip.run_cell(
+        """%%sql --save test_dataset --no-execute
+SELECT *
+FROM data.csv
+"""
+    )
+    out = ip.run_cell(cell)
+    assert out.error_in_exec is error_check
```

Comment:
I don;t think we need this check since there's no error now. Please remove the parametrize also